src/main/java/org/onebusaway/alexa/AuthedSpeechlet.java

```diff
@@ -79,7 +79,7 @@ public SpeechletResponse onIntent(final IntentRequest request,
             return anonSpeechlet.onIntent(request, session);
         } else if (GET_CITY.equals(intent.getName())) {
             return getCity();
-        } else if (SET_STOP_NUMBER.equals(intent.getName())) {
+        } else if (SET_STOP_NUMBER.equals(intent.getName()) || YES.equals(intent.getName()) || NO.equals(intent.getName())) {
```

Comment:
Could you please split out the YES and NO intents to their own `else if` block?

`} else if (YES.equals(intent.getName()) || NO.equals(intent.getName()) {`

For now these are associated with the `SET_STOP_NUMBER` intent, but they could be more general in the future, and you've added code elsewhere to generalize them.